src/components/AuthContext/AuthContext.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react'
```

Comment:
During the last merge, useEffect was erased on this line, and we can`t fix it because deadline has started :(